public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -169,45 +153,17 @@ const ui = {
   },
 };
 
-const server = setupServer();
+const server = setupMswServer();
 
-const configureMockServer = () => {
+const configureMockServer = (server: SetupServer) => {
   mockSearchApi(server).search([]);
   mockUserApi(server).user(getMockUser());
-  mockFolderApi(server).folder(
```

Comment:
Was mostly able to remove these because of the recently skipped tests (pausing + lotex)